tests/ast/data/Dataframe.to_snowpark_pandas.test

```diff
@@ -35,7 +35,10 @@ body {
           }
         }
         src {
+          end_column: 41
```

Comment:
What python version?  I don't think we should need to include line info here as it varies per python version.

Reply:
There's already conditional logic for this.

Reply:
Yeah, when we're verifying the AST this info is ignored but I can get rid of these changes here if you'd like.